todoCLI/todoCLI.js

```diff
@@ -0,0 +1,60 @@
+//----Query Line No 47 --> this inside fsWrite/Read
+
+const fs = require("fs");
+const readline = require("readline");
+const colors = require("colors");
+
+module.exports = class todolist {
+  //---Default Constructor ;
+  constructor(fName) {
+    this.fName = fName;
+    this.arr = [];
+    try {
+      if (fs.existsSync(this.fName)) {
+        let data = fs.readFileSync(this.fName, "utf8");
+        this.arr = JSON.parse(data);
+      }
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  //-----adding item into array ;
+  addItem = function(item) {
+    this.arr.push({ completed: false, title: item });
```

Comment:
Somewhere along the way these functions were put into the body of a class so all of the method definitions need to be updated to the following syntax, otherwise a `SyntaxError` will be thrown:

```
addItem(item) {
    // ...
}
```